spec/lib/diaspora/message_renderer_spec.rb

```diff
@@ -180,7 +180,7 @@ def message(text, opts={})
       end
 
       it "should leave mentions intact for real diaspora handles" do
-        new_person = FactoryGirl.create(:person, diaspora_handle: 'maxwell@joindiaspora.com')
+        new_person = FactoryBot.create(:person, diaspora_handle: 'maxwell@joindiaspora.com')
```

Comment:
Style/StringLiterals: Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.